services/core-api/app/api/now_applications/transmogrify_now.py

```diff
@@ -29,6 +29,12 @@ def code_lookup(model, description, code_column_name):
         result = None
     return result
 
+def get_boolean_value(field):
```

Comment:
👍

Reply:
However, this changes the original logic for boolean field processing. Is it intentional?

Reply:
Yes the original logic ignoring null values, so if the string wasn't "Yes" it was defaulting to false, even if the field was null.